### PR TITLE
click 0.3.2 (new formula)

### PR DESCRIPTION
**Pull request by mkw**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/click.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class Click < Formula
+  desc "The command-line interactive controller for Kubernetes"
+  homepage "https://github.com/databricks/click"
+  url "https://github.com/databricks/click/archive/v0.3.2.tar.gz"
+  sha256 "eed648409bf78a05658a9d097e5099ca17bf19df70122e2067859ae94c5575d5"
+  head "https://github.com/databricks/click.git"
```
Comment (commitay) [inline Formula/click.rb]: Remove head.

File: Formula/click.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  depends_on "rust" => :build
```
Comment (commitay) [inline Formula/click.rb]: New line between `sha256` and `depends_on`

File: Formula/click.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+  def install
+    system "cargo", "build", "--release"
+    bin.install "target/release/click"
+  end
+
+  test do
+    # A better test would requrie a full K8s cluster, unfortunately
+    system bin/"click", "-V"
```
Comment (commitay) [inline Formula/click.rb]: Can we check for an error instead?

**Review by commitay — DISMISSED** [review]

File: Formula/click.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
+class Click < Formula
+  desc "The command-line interactive controller for Kubernetes"
+  homepage "https://github.com/databricks/click"
+  url "https://github.com/databricks/click/archive/v0.3.2.tar.gz"
+  sha256 "eed648409bf78a05658a9d097e5099ca17bf19df70122e2067859ae94c5575d5"
+  head "https://github.com/databricks/click.git"
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "build", "--release"
```
Comment (commitay) [inline Formula/click.rb]: `system "cargo", "install", "--root", prefix` and remove the `bin.install`

File: Formula/click.rb (hunk 3 above)
Comment (mkw) [reply Formula/click.rb]: Still working on this -- need to figure out how to use expect.

Comment (mkw) [reply Formula/click.rb]: I think it's more dangerous to try to check for a non-existent cluster due to too many network dependencies.  Instead, I setup fake configuration files and at least confirm that it starts in the correct, default, state.  Is that good enough?

**Comment by mkw** [comment]
@commitay I believe that I made all the requested changes.  Please let me know if there's anything else that I need to do.

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @mkw for the pull request!

**Pull request closed without merging**